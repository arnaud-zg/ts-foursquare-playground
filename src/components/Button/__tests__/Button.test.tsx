import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from '..'

describe('Component/Button', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Button label="Ok" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
