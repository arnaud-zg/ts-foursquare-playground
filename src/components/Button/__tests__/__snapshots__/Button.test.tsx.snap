// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Button renders correctly 1`] = `
<button
  className=" bg-white text-gray-800 font-bold rounded border-b-2 shadow-md py-2 px-6 inline-flex items-center border-green-500 hover:border-green-600 hover:bg-green-500 hover:text-white"
>
  <span
    className="mr-2"
  >
    Ok
  </span>
</button>
`;
